### ISSUE TITLE
When opening the blazorbook location : "there is nothing 

### ISSUE THREAD

**Issue by paulvanbladel** [aeagle/blazor-book#4]
Looks like a very interesting project.
When I open the blazorbook location, I get the above error (which is simply the "notfound" found route of my router component in the main app).

I installed your nuget in both the webassembly project and the razor class lib (i named it "StoryBook").
I added the "StoryBook" assembly as referenced assembly
and added following line in program.cs
```
builder.RegisterAllStories(Assembly.Load("StoryBook"));
```

What am I missing?

**Comment by aeagle** [comment]
Hi @paulvanbladel. It seems I've missed a steps in the instructions to setup. The following needs to be added:

1) Add `@using BlazorBook.Components` to your `_Imports.razor`
2) Add a `<BlazorBook.Resources />` at the top of `App.razor`
3) Add an `AdditionalAssemblies` prop on the `<Router />` component in `App.razor` like so:

```html
<Router AppAssembly="@typeof(App).Assembly" AdditionalAssemblies="new[] { typeof(BlazorBook.UI).Assembly }">
```

I'll have to see if I can get this setup more automatic when I get a mo. I do seem to have a styling issue with the UI in the beta package I deployed. I'll also look to get a fix in for that.

**Comment by aeagle** [comment]
I've just published a new version - `0.0.1-beta.2`. This should sort the UI styling issue.

**Comment by paulvanbladel** [comment]
Thanks a lot
